inotify.go

```diff
@@ -36,7 +37,7 @@ type Watcher struct {
 // NewWatcher establishes a new watcher with the underlying OS and begins waiting for events.
 func NewWatcher() (*Watcher, error) {
 	// Create inotify fd
-	fd, errno := unix.InotifyInit()
+	fd, errno := unix.InotifyInit1(syscall.IN_CLOEXEC)
```

Comment:
This should be `unix.IN_CLOEXEC` since `unix` is a more up-to-date version of the `syscall` package.